guides/common/modules/con_smart-proxy-networking.adoc

```diff
@@ -0,0 +1,54 @@
+[id="{smart-proxy-context}-networking"]
+= {SmartProxy} networking
+
+The communication between {ProjectServer} and hosts registered to a {SmartProxyServer} is routed through that {SmartProxyServer}.
+{SmartProxyServer} also provides {Project} services to hosts.
+
+Integrated {SmartProxy}::
+Integrated {SmartProxy} is the {SmartProxy} contained in a {ProjectServer}.
+A host that is directly connected to a {ProjectServer} is considered a client of its integrated {SmartProxy}.
```

Comment:
IMHO a host isn't a client of "its integrated {SmartProxy}". Especially in the non-Katello context that's technically false: hosts don't connect to `{SmartProxy}`.